CMakeModules/ODInternal.cmake

```diff
@@ -236,3 +236,47 @@ if ( NOT ${BUILDINSRC} )
 			    PATTERN "*.ico" PATTERN "*.rc" PATTERN "*.txt"
 			    EXCLUDE PATTERN CMakeFiles EXCLUDE )
 endif()
+
+if ( APPLE )
+
+	set(LIBRARY_DESTINATION "${CMAKE_BINARY_DIR}/Contents/Frameworks/debug")
+	file(MAKE_DIRECTORY ${LIBRARY_DESTINATION})
+
+	if (TARGET Freetype::Freetype)
+		get_target_property(FREETYPE_LIBRARY Freetype::Freetype IMPORTED_LOCATION)
```

Comment:
Nowadays one should avoid reading the target properties, and use generator expressions instead, thus lines 246-247 should not be required